staging/src/k8s.io/apiextensions-apiserver/third_party/forked/celopenapi/model/escaping_test.go

```diff
@@ -170,3 +170,36 @@ func TestEscapingFuzz(t *testing.T) {
 }
 
 var validCelIdent = regexp.MustCompile(`^[a-zA-Z_][a-zA-Z0-9_]*$`)
+
+func TestCanSkipRegex(t *testing.T) {
+	cases := []struct {
+		unescaped        string
+		canSkip          bool
+		invalidCharFound bool
+	}{
+		{unescaped: "a.a", canSkip: false},
+		{unescaped: "a-a", canSkip: false},
+		{unescaped: "a__a", canSkip: false},
+		{unescaped: "a.-/__a", canSkip: false},
+		{unescaped: "a_a", canSkip: true},
```

Comment:
`a_a_a` also. The code handles it, I believe. But it's an important corner case.